### ISSUE TITLE
New referrer spammer links to add to blacklist

### ISSUE THREAD

**Issue by Bondmatrix** [piwik/referrer-spam-blacklist#85]
These spam referrer links appeared in my GA account at my domain msinterim.se.  
The first 3 addresses are versions of the already blacklisted floating free share buttons in order to fool the blacklist.Other reasons for that these are spamlinks are a 100% bounce rate, a  weird name and a unusual high session rate and number of users for a non trading website/domain. These are spam links. Thanks for blacklisting these links and making my life easier. 
site4.floating-share-buttons.com
site1.floating-share-buttons.com
site3.floating-share-buttons.com
www.Get-Free-Traffic-Now.com
www1.free-social-buttons.com
www4.free-social-buttons.com
www5.free-social-buttons.com
www.event-tracking.com

**Comment by Vanealma** [comment]
Maybe we should directly ban any website that contains the word "buttons".
Unless you sell buttons, why would any website containing that word link to
you? These button websites are driving me nuts too...

**Comment by Bondmatrix** [comment]
No, the website i have is a consulting website. I do not sell buttons. 100% bounce rate, high number of session and reported users on a non trading website is clearly spam were floating share buttons is already blacklisted.
I started first having semalt problems and all kinds of russian spam links in 2014 and in gradually changed into these links until I got tired of it. 
Non of these links are in anyway connected to me. So if Piwik and git hub community can blacklist the "hell" out of these links, the better for all. I tried to check with previous links in the list. and I did not find the ones I reported in that list. 
Thanks for your support Piwik
